### PR TITLE
Add ability to run a one off data exporter using provided start and e…

### PR DESCRIPTION
**Pull request by hbrace**
### Why?
For the initial export of data to DAP and MI, its gonna be a big one! We need to be able to provide a start and end date to export the data already inserted into the database
### What?
Script checks for start and end date variables and uses these if present to select the data. These should only be present when running manually using the runbook. Also added an echo at the end to state when the export has finished.
### How?
Test manually by following the run book attached to the trello ticket. Also need to use the kubernetes repo checked out on the branch with the same name.
### Links?
https://trello.com/c/Ml32xpgI/1714-runbook-and-script-modify-mi-dap-one-off-initial-export-8

### CODE REVIEW

**Comment by hbrace** [comment]
[Runbook and script modify - MI & DAP one-off initial export (8)](https://trello.com/c/Ml32xpgI/1714-runbook-and-script-modify-mi-dap-one-off-initial-export-8)

File: docker/run.sh [hunk 1]
```diff
@@ -8,11 +8,20 @@ set -o nounset
...
-
 PERIOD_DATE=$(date -u -d@"$(( `date -u +%s`-86400))" "+%Y-%m-%d")
-START_OF_PERIOD=$(date -d@"$(( `date -u +%s`-86400))" "+%Y-%m-%d 00:00:00")
-END_OF_PERIOD=$(date -u "+%Y-%m-%d 00:00:00")
 
+if [ -z "$START_DATE" ]; then
+  START_OF_PERIOD=$(date -d@"$(( `date -u +%s`-86400))" "+%Y-%m-%d 00:00:00")
+else
+  START_OF_PERIOD="$START_DATE 00:00:00"
+  PERIOD_DATE="${START_DATE}_initial_export"
```
Comment (AdamHawtin) [inline docker/run.sh]: I think this should include the end date when it's being set manually.

**Review by AdamHawtin — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]
Looks nice

**Review by NickGrantX — COMMENTED** [review]
Just need to remember that the 'end' of the period is midnight, so doesn't include the end date at all... well except for the picosecond at the stroke of midnight

**Pull request merged**